web/thesauruses/java/control_structures.json

```diff
@@ -0,0 +1,62 @@
+{
+  "meta": {
+    "language": "Java",
+    "language_version": "15",
+    "language_name": "Java"
+  },
+  "categories": {
+    "Conditional Statements (ifs)": [
+      "if_conditional",
+      "if_else_conditional",
+      "if_elseif_conditional",
+      "if_elseif_else_conditional",
+      "ternary conditional"
+    ],
+    "Loops": [
+      "while_loop",
+      "do_while_loop",
+      "for_loop"
+    ],
+    "Iterations": [
+      "foreach_loop"
+    ]
+  },
+  "control_structures": {
+    "if_conditional": {
+      "name": "If conditional",
+      "code": "if(condition){code block to be executed}"
```

Comment:
For things like `code block to be executed`, could you make it look like a real function? Or leave a comment that that's where it will be? I just want to make sure it's not confusing English parts with code parts.

And then you could show that things can be on new lines with `\n` in the strings as well (which I know is grammatically correct for Java as-is but offers readability).

And same for other ifs/loops/etc. below.